src/containers/MyNdla/MyNdlaLayout.tsx

```diff
@@ -178,6 +179,14 @@ const MyNdlaLayout = () => {
                   </TreeStructureWrapper>
                 )}
               </StyledLi>
+              <StyledLi role="none">
+                <NavigationLink
+                  id="arena"
+                  name="Arena"
+                  shortName="Arena"
+                  icon={<ForumOutlined />}
+                />
+              </StyledLi>
```

Comment:
Denne bør wrappes i en sjekk om brukeren har arenaEnabled som felt. Det betyr at du må legge inn en usePersonalData() her for å finne det.